tests/test_validation/config_validation_test.py

```diff
@@ -0,0 +1,42 @@
+import sys
+sys.path.insert(0, 'src/')
+from validator import config_validator as validator 
+import json
+
+
+def test_record_count():
```

Comment:
put comments on each test to describe what it does

Reply:
done